### PR TITLE
[VSLCV] dynamic image size

### PR DESCRIPTION
**Pull request by srallen**
_Please request review from `@zooniverse/frontend` team. If PR is related to design, please request review from `@beckyrother` in addition._ 

Package: lib-classifier

Describe your changes:
Fixes the image displayed in the VSLCV to be dynamically sized reusing the container component and its onload behavior now that the store connection is separate. ~This is currently branched off of `single-image-viewer-connector` and will update to merge into master once that's merged.~

Also updates the figure caption since `temperature` was redundant.

This can be reviewed using the dev classifier https://localhost:8080/?project=4878&workflow=16135&env=production at or the project app at: http://localhost:3000/projects/adamamiller/stellar-sleuths/classify

Here's a screenshot:

Screen Shot 2020-09-24 at 1 34 16 PM

Note that the overall height of the viewer area is solved in #1795 

# Review Checklist

## General

- [x] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?

## Components
- [ ] Has a storybook story been created or updated?
- [ ] Is the component accessible? 
  - [ ] Can it be used with a screen reader? [BBC guide to testing with VoiceOver](https://bbc.github.io/accessibility-news-and-you/accessibility-and-testing-with-voiceover-os.html)
  - [ ] Can it be used from the keyboard? [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - [ ] Is it passing accessibility checks in the storybook?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [x] Can you `yarn panic && yarn bootstrap` or `docker-compose up --build` and app works as expected?

## Publishing

- [ ] Is the changelog updated?
- [ ] Are the dependencies updated for apps and libraries that are using the newly published library?

## Post-merging

- [ ] Did the app deploy to https://frontend.preview.zooniverse.org/projects/:project-name/:owner or https://frontend.preview.zooniverse.org/about?
- [ ] Is the new feature working or bug now fixed?
  - [ ] Is there a Talk or blog post written to announce the new feature(s)?
- [ ] Is the design working across browsers (Firefox, Chrome, Edge, Safari) and mobile?
  - [ ] Is this approved by our designer?
- [ ] Is this ready for production deployment?

### CODE REVIEW

**Comment by srallen** [comment]
Ok, so the overall height of the subject viewer is fixed and the sizing of the image is now dynamic:

Screen Shot 2020-09-28 at 3 05 34 PM

This particular image isn't the full height because its aspect ratio is 1:1, so it's a square. We can't really make it fit without weird stretching. @beckyrother I could center it?:

Screen Shot 2020-09-28 at 3 05 24 PM

Or we can tell the team that the image shouldn't be 1:1 if possible. Do you have an idea of what the ratio would be here? 9:16?

**Review by mcbouslog — APPROVED** [review]
This LGTM, tested on projects noted as well as other test workflows using SingleImageViewer (including multi-image subjects) without issue.

File: packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleImageViewer/SingleImageViewerContainer.js [hunk 1]
```diff
@@ -57,7 +57,9 @@ class SingleImageViewerContainer extends React.Component {
   async preload () {
     const { subject } = this.props
     if (subject && subject.locations) {
+      console.log('subject', subject)
```
[the remark this answers is not in the document]
Comment (srallen) [reply packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleImageViewer/SingleImageViewerContainer.js:60]:
```suggestion
```

File: packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleImageViewer/SingleImageViewerContainer.js [hunk 2]
```diff
@@ -57,7 +57,9 @@ class SingleImageViewerContainer extends React.Component {
...
       const imageUrl = Object.values(subject.locations[0])[0]
+      console.log('imageUrl', imageUrl)
```
Comment (srallen) [inline packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleImageViewer/SingleImageViewerContainer.js:62]:
```suggestion
```

**Comment by srallen** [comment]
I'm going to merge this because the research team wants to take screenshots for the tutorial soon and I want the image at least to have the improved label deployed. If we want to do it, the centering of the image can be done in a separate PR and the aspect ratio question is independent of the code.

**Pull request merged**